test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secret2 *api.Secret) {
```

Comment:
`secret2` doesn't have any meaning for this function, really. It seems cleaner if it just can take the secret name to use as parameter, instead of an `api.Secret`.

Does it make sense to you?